migrations/versions/5c21ad23b0f7_.py

```diff
@@ -20,12 +20,13 @@ def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.create_table('performance_profile',
     sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('host_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('inventory_id', postgresql.UUID(as_uuid=True), nullable=False),
     sa.Column('performance_record', postgresql.JSONB(astext_type=sa.Text()), nullable=True),
     sa.Column('performance_score', postgresql.JSONB(astext_type=sa.Text()), nullable=True),
-    sa.Column('report_date', sa.DateTime(), nullable=True),
+    sa.Column('report_date', sa.Date(), nullable=True),
     sa.PrimaryKeyConstraint('id'),
-    sa.UniqueConstraint('host_id')
+    sa.UniqueConstraint('inventory_id'),
```

Comment:
Can't we define a single unique constraint on these two columns?
For now if we can handle logic of host update on processor level then I think it is better to declare combined key? 
what do you think?

Reply:
You are right.  Obviously, Report_date cannot be unique. A unique constraint should be on two columns `inventory_id` and `report_date`.

Changing the code so DB schema will look like below. 
```
postgres=# \d performance_profile;
                                  Table "public.performance_profile"
       Column       |  Type   | Collation | Nullable |                     Default                     
--------------------+---------+-----------+----------+-------------------------------------------------
 id                 | integer |           | not null | nextval('performance_profile_id_seq'::regclass)
 inventory_id       | uuid    |           | not null | 
 performance_record | jsonb   |           |          | 
 performance_score  | jsonb   |           |          | 
 report_date        | date    |           |          | 
Indexes:
    "performance_profile_pkey" PRIMARY KEY, btree (id)
    "performance_profile_inventory_id_key" UNIQUE CONSTRAINT, btree (inventory_id)
    "performance_profile_inventory_id_report_date_key" UNIQUE CONSTRAINT, btree (inventory_id, report_date)
```